### PR TITLE
Remove SVB wording

### PR DESCRIPTION
**Pull request by YodaLightsabr**

### CODE REVIEW

File: components/bank/first/features.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ export default function Features() {
       >
         <Text variant="caption" sx={{ color: 'muted' }}>
           Hack Club does not directly provide banking services. Banking services
-          provided by Silicon Valley Bank, an FDIC-certified institution.
+          provided by another FDIC-certified financial institution.
```
Comment (maxwofford) [inline components/bank/first/features.js]:
```suggestion
          provided by FDIC-certified financial institutions.
```

File: components/bank/start.js [hunk 2]
```diff
@@ -100,9 +100,8 @@ export default function Start() {
           }}
         >
           <Text sx={{ fontSize: 18, color: 'muted', mx: [3, null, 6] }}>
-            Hack Club does not directly provide banking services. Banking
-            services provided by Silicon Valley Bank, an FDIC-certified
-            institution.
+            Hack Club does not directly provide banking services. Banking services
+            provided by another FDIC-certified financial institution.
```
Comment (maxwofford) [inline components/bank/start.js]:
```suggestion
            provided by FDIC-certified financial institutions.
```

**Review by maxwofford — APPROVED** [review]
Added 2 wording changes. Commit them, then please merge!

**Comment by YodaLightsabr** [comment]
@maxwofford I don’t have write access on main for the site repo

**Pull request merged**